src/main/app/common/router/routablePath.ts

```diff
@@ -0,0 +1,34 @@
+const pathParameterRegex = /\/:[^/]+/g;
+
+export class RoutablePath {
+  private _uri: string
+
+  constructor (uri: string, public feature: boolean = true) {
+    if (!uri || !uri.trim()) {
+      throw new Error('uri is missing');
+    }
```

Comment:
Test this constructor for quick increase of code coverage.

Reply:
done